base/src/main/java/uk/ac/ebi/atlas/model/card/CardModelFactory.java

```diff
@@ -21,15 +22,29 @@ public CardModelFactory(UrlHelpers urlHelpers) {
         this.urlHelpers = urlHelpers;
     }
 
-    public CardModel create(PopularSpeciesInfo popularSpeciesInfo) {
+    private CardModel createCardModel(PopularSpeciesInfo popularSpeciesInfo, CardIconType type, List content){
         return CardModel.create(
-                CardIconType.SPECIES,
+                type,
                 popularSpeciesInfo.species(),
                 Pair.of(StringUtils.capitalize(popularSpeciesInfo.species()),
                         Optional.of(urlHelpers.getExperimentsFilteredBySpeciesUrl(popularSpeciesInfo.species()))),
-                Collections.singletonList(
-                        Pair.of(popularSpeciesInfo.totalExperiments() + " experiments",
-                                Optional.empty())));
+                content);
+    }
+
+    public CardModel create(PopularSpeciesInfo popularSpeciesInfo) {
+        List<Pair<String, Optional<String>>> content = new ArrayList<>();
+        content.add(Pair.of(popularSpeciesInfo.totalExperiments() + " experiments",
+                Optional.empty()));
+        if(popularSpeciesInfo.baselineExperiments() > 0){
+            content.add(Pair.of("Baseline: " + popularSpeciesInfo.baselineExperiments(),
+                Optional.of(urlHelpers.getExperimentsFilteredBySpeciesAndExperimentType(popularSpeciesInfo.species(), "baseline"))));
+        }
+        if(popularSpeciesInfo.differentialExperiments() > 0){
+            content.add(Pair.of("Differential: " + popularSpeciesInfo.differentialExperiments() + " ",
+                Optional.of(urlHelpers.getExperimentsFilteredBySpeciesAndExperimentType(popularSpeciesInfo.species(), "differential"))));
+        }
+
+        return createCardModel(popularSpeciesInfo, CardIconType.SPECIES, content);
```

Comment:
You only call this private `createCardModel` method once. Was the intention to call it from `createLandingPageImageCard` as well?